exporter/awscloudwatchlogsexporter/config.go

```diff
@@ -42,6 +40,9 @@ type Config struct {
 	// e.g. logs.us-east-1.amazonaws.com
 	// Optional.
 	Endpoint string `mapstructure:"endpoint"`
+
+	exporterhelper.RetrySettings `mapstructure:"retry_on_failure"`
+	exporterhelper.QueueSettings `mapstructure:"sending_queue"`
```

Comment:
```suggestion
	QueueSettings `mapstructure:"sending_queue"`
}

// NB: exporterhelper.QueueSettings cannot be use beucase on QueueSize is user-settable
type QueueSettings struct {
	QueueSize `mapstructure:"sending_queue"`
}
```
Maybe this is what we want here actually?